demo/src/pages/Workout.tsx

```diff
@@ -527,13 +529,64 @@ const Workout = ({ history }) => {
     isPreJoin,
   };
 
+  const [connectionDetails, setConnectionDetails] = useState() as any;
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get('https://ipapi.co/json/')
```

Comment:
I tried that URL on my laptop at office, got this data:
{
    "ip": "209.249.5.130",
    "network": "209.249.0.0/21",
    "version": "IPv4",
    "city": "Rockville Centre",
    "region": "New York",
    "region_code": "NY",
    "country": "US",
    "country_name": "United States",
    "country_code": "US",
    "country_code_iso3": "USA",
    "country_capital": "Washington",
    "country_tld": ".us",
    "continent_code": "NA",
    "in_eu": false,
    "postal": "11570",
    "latitude": 40.6598,
    "longitude": -73.6371,
    "timezone": "America/New_York",
    "utc_offset": "-0400",
    "country_calling_code": "+1",
    "currency": "USD",
    "currency_name": "Dollar",
    "languages": "en-US,es-US,haw,fr",
    "country_area": 9629091.0,
    "country_population": 327167434,
    "asn": "AS6461",
    "org": "ZAYO-6461"
}

City is not correct. So lets dump the entire json to DB.